conf/server.yaml.template

```
@@ -8,6 +8,8 @@ SERVER:
     # RELEASE: populate with satellite version
     # The snap version currently testing (if applicable)
     # SNAP:
+    # The base os rhel version where the satellite installed
+    # RHEL_RELEASE: 7.9
```

I would keep consistent naming here.
```suggestion
    # RHEL_VERSION: 7.9
```

---

Updated as suggested.

---

@shweta83 Please enclose the value for `RHEL_VERSION` with quotes the same way you did it in `conf/robottelo.yaml.template`. String type should protect us from `6.10` being `6.1`. I know it is just a template, but people frequently copy the templates.

---

Updated.